3rdparty/sgxsdk/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+if (OE_SGX)
+  # This 3rdpary directory only contains header files for compilation of
+  # SGX-specific host-side code.
+  # There is no new artifacts (e.g. new targets) to be added.
```

Consider changing this to a README.md file instead (see other 3rdparty folders) since this only contains a comment for the folder.

Also some typo and grammar corrections:
```suggestion
  # This 3rdparty directory only contains header files for compiling
  # SGX-specific host-side code.
  # There are no new artifacts (e.g. new targets) to be added.
```

---

Thanks!